src/content/en/tools/chrome-devtools/beginners/devtools-for-beginners1.md

```diff
@@ -0,0 +1,239 @@
+  </figcaption>
+</figure>
+
+3. Click **Remix**. Glitch creates a copy of the project that you can 
+edit. Note that the name of the new project will be randomly generated
+and not dfb-1.
+The content is the same, but the name on the top-left has changed.
+4. Click Show Live. Another tab opens with view of what your site 
+currently looks like.
+
+<figure> <img src="2tabs.png" alt="An image of the two open tabs" width="auto" height="auto">
+<figcaption>
+    <b>Figure 3</b>. The viewing and editing tabs.
+  </figcaption>
+</figure>
+
+Now, you have two tabs open: the code
+(which will be called the editing tab) 
+and the preview of your website (which will be called the viewing tab). 
+
+## Adding Content {: #adding-content}
+
+Since your website is pretty empty, let’s add some content to it! 
+In the editing tab, you’ll see two sections in your HTML document: 
+the head (the text between `<head>` and `</head>`) and the body 
+(the text between `<body>` and `</body>`). 
+The head contains the metadata of your site, which is invisible 
+data your web browser parses to understand the content of your site. 
+This helps search engines find your pages, among other things. 
+The body contains the actual content of your page, like text or images.
+You won’t be adding much in the `<head>` section in this part of the 
+tutorial, so for now, let’s move on to the fun part, the `<body>` 
+section. Since this is a personal website, you’ll definitely want to 
+have something about yourself in there. Many personal websites have an
+‘About Me’ section, and yours will as well. Here’s how you’ll do it:
+
+1. Go to line 19 and press <kbd>Enter</kbd>
+2. You want a heading for this section, 
+so type `<h1>About Me</h1>`. 
+This formats the text as a heading. 
+3. To see the changes, go to the viewing tab.
+4. Go back to the editing tab.
+5. You’ll want to put in text, too, so insert a `<p>` element, 
+like `<p> I am learning HTML.</p>` under the line where you put 
+the heading. 
+6. Go to the viewing tab.
+7. Go back to the editing tab.
+8. Add a list under the `<p>` element on line 21 that 
+details your accomplishments, like so: 
+	
+  <ul>
+    <li>Developer</li>
+	<li>Tools</li>
+	<li>Search Engine</li>
+  </ul>
+	
+9. Again, 
+go to the viewing tab to see the new content.
+
+## Improving Workflow With DevTools {: #improving-workflow}
+
+At this point, 
+you may have noticed that the process of changing the HTML on your site
+can be somewhat tedious. Wouldn’t it be great if there was an easier way 
+to do this? 
+
+Enter DevTools.
+
+### A Note on Nodes {: #nodes}
+
+ When you open the Elements Panel in DevTools, 
+ you’ll see a screen that looks quite similar to the HTML document
+ you’ve been working on in the editing tab. 
+ However, these are not HTML elements, but are actually DOM nodes.
+ DOM is an interface that represents HTML elements in your browser, 
+ and while the DOM Tree you see in the Elements Panel looks quite 
```

Comment:
A little bit bumpy, and run-on-- this sentence. How about something like:

DOM is an interface that represents HTML elements in your browser. While the DOM Tree you see in the Elements Panel looks quite similar to your HTML document, there are ways you can edit it so that it is not: